crates/wkg/src/main.rs

```diff
@@ -19,18 +23,22 @@ struct Cli {
 #[derive(Args, Debug)]
 struct RegistryArgs {
     /// The registry domain to use. Overrides configuration file(s).
-    #[arg(long = "registry", value_name = "DOMAIN")]
+    #[arg(long = "registry", value_name = "WKG_DOMAIN")]
```

Comment:
It looks like you were maybe thinking the `value_name` here was an `env`?

Reply:
Whoops that was definitely me rebasing poorly